### ISSUE TITLE
Mean Pooling Not Consistent with `embed_all` outputs

### ISSUE THREAD

**Issue by kheyer** [huggingface/text-embeddings-inference#278]
### System Info

Running the `ghcr.io/huggingface/text-embeddings-inference:1.2` container

Info output:

```
{"model_id":"entropy/roberta_zinc_480m","model_sha":null,"model_dtype":"float16","model_type":{"embedding":{"pooling":"mean"}},"max_concurrent_requests":512,"max_input_length":510,"max_batch_tokens":163840,"max_batch_requests":null,"max_client_batch_size":10000,"auto_truncate":false,"tokenization_workers":32,"version":"1.2.3","sha":"cc1c510e8d8af8447c01e6b14c417473cf2dfda9","docker_label":"sha-cc1c510"}
```

### Information

- [X] Docker
- [ ] The CLI directly

### Tasks

- [X] An officially supported command
- [ ] My own modifications

### Reproduction

The following code reproduces the issue:

```python
import torch
import httpx

embedding_url = 'http://localhost:8100'

inputs = {'inputs' : ['Brc1cc2c(NCc3ccccc3)ncnc2s1']}

raw_outputs = torch.tensor(httpx.post(f'{embedding_url}/embed_all', json=inputs).json())
raw_pooled = raw_outputs.mean(1)

pooled_outputs = torch.tensor(httpx.post(f'{embedding_url}/embed', json=inputs).json())

print(raw_pooled)
print(pooled_outputs)

ratio = pooled_outputs/raw_pooled

print(ratio.mean())
```

### Expected behavior

From the `ratio` value printed by the reproduction code, embeddings from the `/embed_all` endpoint, when manually pooled, are about an order of magnitude different compared to the embeddings from `/embed`. The request is a single item, so pad tokens shouldn't be a factor here.

Am I misunderstanding what mean pool means in this context?

**Comment by kheyer** [comment]
Edit: this was the `normalize` flag
